client/scss/components/_preview-panel.scss

```diff
@@ -19,12 +19,29 @@
   }
 
   &__wrapper {
+    position: relative;
     width: calc(var(--preview-iframe-width) * var(--preview-width-ratio));
     height: 100%;
     margin-inline-start: auto;
     margin-inline-end: auto;
   }
 
+  // Transparent overlay to prevent drag-resize issues
+  // when the cursor is above the iframe.
```

Comment:
Just checking if you tried something as simple as setting the iframe to `pointer-events: none` as well?

Reply:
Yes and it didn't work, but we don't need this with the new implementation that doesn't use the Drag API.